plutus-ledger-api/test-plugin/Spec/Data/Budget/9.6/currencySymbolValueOf.budget.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 27490443
+| mem: 64380})
```

Comment:
Non-data version:
```
({cpu: 26667308
| mem: 102924})
```

Reply:
That's a surprisingly negligible difference in CPU units. Well, at least MEM is 1.6x lower.